device-types/HPE/ProCurve-2910-48G.yaml

```diff
@@ -0,0 +1,112 @@
+---
+manufacturer: HPE
+model: ProCurve 2910al-48G
+slug: hpe-procurve-2910al-48g
+part_number: J9147A
+u_height: 1
+comments: '[ProCurve 2910al-48G Specs](https://support.hpe.com/hpesc/public/docDisplay?docId=emr_na-c01844343)'
+is_full_depth: false
+console-ports:
+  - name: Console
+    type: rj-45
+power-ports:
```

Comment:
I dont use these so I do not know, but it appears that the J9147A has a redundant 12V DC port for backup power. If this is true, please add that as well

Reply:
Was just sent a picture of the device, asking about the 12V DC, I'll update the PR when I get confirmation.